src/frontend/src/tests/TestServerData.tsx

```diff
@@ -28,11 +28,9 @@ export const TestFrappes: Array<Frappe> = [
     ],
     "price": 6.1,
     "final_price": 0.0,
-    "menu_key": 3,
```

Comment:
We don't want to change anything in TestServerData.tsx. Aka: this PR is trying to undo the changes I just made to this file.

TestServerData shouldn't really matter at all but we have at least 1 file (I think) that still relies on this and so it will break the frontend without these changes so it's best to just keep these changes as is for now